tests/testpioasm.py

```diff
@@ -44,6 +44,13 @@ def testNop(self):
         self.assertAssemblesTo(".side_set 1\nnop side 1", [0b101_10000_010_00_010])
         self.assertAssemblesTo(".side_set 1\nnop side 1 [1]", [0b101_10001_010_00_010])
 
+    def testSidesetOpt(self):
+        self.assertAssemblesTo(".side_set 1 opt\nnop side 1", [0b101_11000_010_00_010])
```

Comment:
Is it worth having a test for side 0 without the timing option too? Presumably with the 0b101_10000_010_00_010

Reply:
Can't hurt. Added!